api/src/main/java/io/strimzi/api/kafka/model/kafka/entityoperator/EntityOperatorSpec.java

```diff
@@ -58,6 +59,8 @@ public void setUserOperator(EntityUserOperatorSpec userOperator) {
         this.userOperator = userOperator;
     }
 
+    @Deprecated
+    @DeprecatedProperty(description = "TLS sidecar was removed in Strimzi 0.41.0.")
```

Comment:
In the `CHANGELOG` you have that it's deprecated and it will be ignored.
Should here be the "deprecated" statement, instead of "removed", as well?

Reply:
Yep, thanks.